bin/opteadm/src/bin/opteadm.rs

```diff
@@ -220,24 +224,45 @@ enum Command {
 
     /// Add a new router entry, either IPv4 or IPv6.
     AddRouterEntry {
-        /// The OPTE port to which the route is added
-        #[arg(short)]
-        port: String,
-        /// The network destination to which the route applies.
-        dest: IpCidr,
-        /// The location to which traffic matching the destination is sent.
-        target: RouterTarget,
+        #[command(flatten)]
+        route: RouterRule,
+    },
+
+    /// Remove an existing router entry, either IPv4 or IPv6.
+    DelRouterEntry {
+        #[command(flatten)]
+        route: RouterRule,
     },
 
     /// Configure external network addresses used by a port for VPC-external traffic.
     SetExternalIps {
-        /// The OPTE port to which the route is added
+        /// The OPTE port to configure
         #[arg(short)]
         port: String,
 
         #[command(flatten)]
         external_net: ExternalNetConfig,
     },
+
+    /// Allows a guest to send and receive traffic on a given CIDR block.
```

Comment:
I can only really think of contrived scenarios and like you said this is already a bit more fine-grained than what the RFD mentions, but is it worth allowing just a given direction?

(not a blocker)

Reply:
That makes sense to me -- it's not a complex change and it lets us kick 'rule insert rollback' down the road for a little longer.

Done.